Tests/BasicPerformanceTests/SortedArrayPerfTests.swift

```diff
@@ -11,14 +11,44 @@
 import TestSupport
 import Basic
 
+private let magnitude = 5000
+
+private func prng(_ seed: Int) -> Int {
```

Comment:
For the same input sequence you will get the same output sequence. The sequence is stored so it's also constant. Mostly though it should produce an output sequence which is not trivially sortable.